app/views/clients/rollup/_custom_services.html.haml

```diff
@@ -0,0 +1,13 @@
+:ruby
+  custom_services = @client.custom_services.
+    joins(enrollment: :client).
+    joins(custom_service_type: :custom_service_category).
+    merge(GrdaWarehouse::Hud::Enrollment.visible_to(current_user)).
+    preload( :enrollment, :project, :client, :custom_service_type ).
+    preload(custom_service_type: :custom_service_category).
+    order(date_provided: :desc)
```

Comment:
Maybe this should be a method on Client (or a class method on CustomService) something like `services_for_rollup` This is a lot of logic to hide in the view.